config.default

```diff
@@ -33,6 +33,16 @@ referenceRunName = None
 # MPAS-Analysis)
 preprocessedReferenceRunName = None
 
+[execute]
+## options related to executing parallel tasks
+
+# the number of parallel tasks (1 means tasks run in serial, the default)
+parallelTaskCount = 1
+
+# Prefix on the commnd line before a parallel task (e.g. 'srun -n 1 python')
+# Default is no prefix (run_analysis.py is executed directly)
+commandPrefix = 
```

Comment:
I should add a script for submitting parallel analysis jobs on each machine.  The script could handle making a little config file that sets the `commandPrefix` and maybe also the output directory.